packages/client/src/components/auth/PrivateRoute.tsx

```diff
@@ -22,23 +23,29 @@ const PrivateRoute: React.FC<RouteProps> = (props) => {
   const isAuthEmpty = useSelector(getIsAuthEmpty);
   const isAdmin = useSelector(getIsAdmin);
   const isAuthLoaded = useSelector(getIsAuthLoaded);
+  const secretKey = useSelector(getBookingsSecretKey);
 
   switch (true) {
-    // display loading state until initial auth is loaded
+    // Display loading state until initial auth is loaded
```

Comment:
Nitpick (and not related to the work in this PR): all cases in this `switch` block end with a `return` statement. The `switch` block could be removed and the `case` blocks could become `if`s. It would be more readable this way IMO. No need to change it, it's a small thing, but I wanted to share my thoughts about it.